wfc3tools/wf3rej.py

```diff
@@ -89,19 +61,17 @@ def wf3rej(input, output="", crrejtab="", scalense="", initgues="",
     ----------
     input : str or list
         Name of input files, such as
-        - a single filename (``iaa012wdq_raw.fits``)
         - a Python list of filenames
-        - a partial filename with wildcards (``*raw.fits``)
-        - filename of an ASN table (``*asn.fits``)
+        - a partial filename with wildcards (``*flt.fits``)
```

Comment:
just clarifying the change here - this function does not take in associations anymore?

Reply:
This function actually could never process ASN tables as the underlying C code does not support this feature.